starfish/imagestack/indexing_utils.py

```diff
@@ -26,8 +27,42 @@ def convert_to_selector(
     return return_dict
 
 
-def index_keep_dimensions(
-        data: xr.DataArray, indexers: Mapping[str, Union[int, slice]]) -> xr.DataArray:
+def convert_coords_to_indices(array: xr.DataArray,
+                              indexers: Mapping[Coordinates, Union[Number, Tuple[Number, Number]]]
+                              ) -> Dict[Axes, Union[int, tuple]]:
```

Comment:
```suggestion
                              ) -> Mapping[Axes, Union[float, Tuple[Number, Number]]]:
```

Reply:
this method can't return a float since we're converting to positional indexers.